clients/drcachesim/tracer/kcore_copy.cpp

```diff
@@ -79,8 +76,93 @@ struct proc_kcore_code_segment_t {
     char *buf;
 };
 
-kcore_copy_t::kcore_copy_t()
-    : modules_(nullptr)
+/* The auto close wrapper of file_t.
+ * This can ensure the file is closed when it is out of scope. And it is also allowed to
+ * cutomize file manipulation functions.
+ */
+struct file_autoclose_t {
+    file_t fd;
```

Comment:
As a `class` these member fields should have end in `_`

Reply:
Done.